core/src/main/java/org/hibernate/ogm/dialect/impl/EmptyMultiTableBulkIdStrategy.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.dialect.impl;
+
+import org.hibernate.boot.spi.MetadataImplementor;
+import org.hibernate.boot.spi.SessionFactoryOptions;
+import org.hibernate.engine.jdbc.connections.spi.JdbcConnectionAccess;
+import org.hibernate.engine.jdbc.spi.JdbcServices;
+import org.hibernate.engine.spi.SessionFactoryImplementor;
+import org.hibernate.hql.internal.ast.HqlSqlWalker;
+import org.hibernate.hql.spi.id.MultiTableBulkIdStrategy;
+
+public class EmptyMultiTableBulkIdStrategy implements MultiTableBulkIdStrategy {
```

Comment:
I can see that this is working but I would like to understand why. A bit of description would be helpful

Reply:
Ok, I'll see deeper. But in the first look, MultiTableBulkIdStrategy is used while multi tables updates or deletions. But in my case there is single table.

Reply:
After looking at it I realise that we don't create  in muliple tables in OGM for the same entity.
The reason is because we don't have join for all the datastores. Still, not everybody i sfamiliar with the codebase and there should be some javadoc in places where we have this kind of magic.

Reply:
What do you mean?